src/MfaVerifyController.js

```diff
@@ -127,6 +127,15 @@ function (Okta, Checkbox, BaseLoginController, CookieUtil, TOTPForm, YubikeyForm
         }
       }
       return false;
+    },
+
+    back: function() {
+      // Empty function on verify controllers to prevent users
+      // from navigating back during 'verify' using the browser's
+      // back button. The URL will still change, but the view will not
+      // We will need to think of a better solution for this, possibly
```

Comment:
not sure we need to include the part of what we could do in the future. I would leave it out
@rchild-okta thoughts?